test/type/opaque_with_conditional_availability.swift

```diff
@@ -0,0 +1,108 @@
+// RUN: %target-swift-frontend -target x86_64-apple-macosx10.15 -typecheck -verify %s
```

Comment:
Would this have the same problem as the `spi-available-inline` and fail running locally on M1 macs? Because swift-frontend target will try load x86_64 stdlib and fail, so should we use dynamic target as well `%target-cpu-apple-macosx10.15`?

Reply:
Yeah, I guess that's reasonable to do...